src/Fixer/Operator/TernaryToNullCoalescingFixer.php

```diff
@@ -91,7 +43,7 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
     public function getDefinition()
     {
         return new FixerDefinition(
-            'Use null coalescing operator `??` wherever possible.',
+            'Use `null` coalescing operator `??` where possible.',
```

Comment:
I see, just checked the readme and currently we got 2 x '\`null\`' and 1x 'null' so this can go both ways :)